### DISCUSSION TITLE
Golang: build tags: how to switch between them?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#4570] [Q&A] [answered]
Currently Coc does not works for the files with the first line like

`//go:build integ_tests`

Do I need to switch somehow manually?

**Comment by fannheyward** [comment]
Needs the language server `gopls` supports this, coc.nvim is a LSP client.

**Reply by vitaly-zdanevich** [reply]
Looks like I found related issue https://github.com/golang/go/issues/29202

**Reply by vitaly-zdanevich** [reply]
@chemzqm still no solution? No way to handle all build tags? On my project I have no linting for tests :(

**Comment by vitaly-zdanevich** [comment, marked as answer]
SOLVED by adding this to the env:

`export GOFLAGS='-tags=unit_tests'`
